static/js/components/inputs/util_test.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { assert } from 'chai';
+
+import { dialogActions } from './util';
+
+describe('input functions', () => {
+
+  describe('dialogActions', () => {
+    it('renders properly', () => {
+      let onCancel = () => null;
+      let onSave = () => null;
+      let inFlight = true;
+      let newClass = "new-class";
+      let newText = "New Text";
+      let actions = dialogActions(onCancel, onSave, inFlight, newText, newClass);
+      assert.lengthOf(actions, 2);
+      let [cancelButton, saveButton] = actions;
+      assert.equal(cancelButton.props.className, "secondary-button cancel-button");
+      assert.equal(cancelButton.props.onClick, onCancel);
+      assert.equal(cancelButton.props.children, "Cancel");
+
+      assert.equal(saveButton.props.className, `primary-button save-button ${newClass}`);
+      assert.equal(saveButton.props.onClick, onSave);
+      assert.equal(saveButton.props.children, newText);
+      assert.equal(saveButton.props.spinning, inFlight);
+      assert.equal(saveButton.type.name, "SpinnerButton");
```

Comment:
do we get any value from asserting the name of the component here? i think the asserting of `spinning` is enough of an indicator that this is supposed to be a button that 'spins', and i don't know if i like the precedent of asserting component class names in general